src/community/leiden.c

```diff
@@ -960,6 +960,9 @@ static igraph_error_t igraph_i_community_leiden(
  * \param start Start from membership vector. If this is true, the optimization
  *    will start from the provided membership vector. If this is false, the
  *    optimization will start from a singleton partition.
+ * \param n_iterations Iterate the core Leiden algorithm for the indicated number 
+ *    of times. If this is a negative number, it will continue iterating until 
+ *    the quality no longer changed.
```

Comment:
```suggestion
 *    the quality no longer changes.
```

Reply:
This is intentionally past tense: it is only the last iteration that didn't change. A next iteration might still change. Perhaps I can better rephrase this to "improved" though.

Reply:
"It continues iterating until the quality stops changing" then? Can a simple past ("changed") follow a future tense ("will continue")?